README.md

```diff
@@ -255,6 +255,8 @@ Follow these steps to build your app for Oculus Quest:
     `../Builds/OculusMobile_Release_OpenBrush_FromGui/`.
 1.  Run `adb install com.Icosa.OpenBrush.apk`.
 
+**Note:** Add your new scene files' names in List '`scene`' on 1221st line in `../Assets/Editor/BuildTiltBrush.cs` before building. If you didn't, your app won't be built with those scenes even if they are put on `Scenes In Build` in `Build Settings`.
```

Comment:
I would avoid mentioning specific line numbers as it's unlikely people will remember to update that if they make changes. Maybe say something like  "... add to the list defined in the DoBuild method ( string[] scenes = {...} )"